xml/dhcp_vendor_class.xml

```diff
@@ -518,6 +518,15 @@
     <param pos="0" name="os.vendor" value="Cisco"/>
   </fingerprint>
 
+  <fingerprint pattern="^Cisco Telepresence Peripheral$">
+    <description>Cisco Telepresence Peripheral</description>
```

Comment:
Minor: update the casing in the description. I understand the DHCP vendor class value has the above casing, however, from a scan of this Google search (`Cisco Telepresenc site:cisco.com`) it appears that Cisco uses "Cisco TelePresence".

Reply:
Done.